jaggeryapps/admin/templates/header.jag

```diff
@@ -62,6 +68,156 @@ try {
     log.error('error occurred while reading site.json' + e);
 }
 
+var billing = billing || (function () {
+
+    var isTrial;
+    var accountContent;
+    var subscription = "identity_cloud";
+
+    var isTrailExpired = function () {
+        return isExpired;
+    };
+
+    var isTrialAccount = function () {
+        return isTrial;
+    };
+
+    var getAccountContent = function () {
+        return accountContent;
+    };
+
+    var getSubscription = function () {
+        return subscription;
+    };
+
+    var initBillingInfo = function () {
+
+    var headerConfig  = getJsonFromFile("/conf/urls.json");
+    var billingStatusFirstLoginUrl = headerConfig.headerUrls.billingStatusFirstLoginUrl;
+    var billingLoginUrl = headerConfig.headerUrls.billingLoginUrl;
+    var cloudMgtUrl = headerConfig.headerUrls.cloudMgtUrl;
+    var cloudMgtIndexPage = cloudMgtUrl + "site/pages/index.jag";
+    var cloudAdminUserName = headerConfig.userInfo.cloudAdminUserName;
+    var cloudAdminPassword = headerConfig.userInfo.cloudAdminPassword;
+
+    var billingResult = {};
+
+    var addDays = function (date, days){
+        var cal = Packages.java.util.Calendar.getInstance();
+        cal.setTime(date);
+        cal.add(Packages.java.util.Calendar.DATE, days);
+        return cal.getTime();
+    };
+
+    var data = {
+        "action": "login",
+        "userName": cloudAdminUserName,
+        "password": cloudAdminPassword
+    };
+
+ try{
+
+    var loginResult = post(billingLoginUrl, data, {}, 'json');
+
+    if(loginResult && loginResult.data) {
+
+        var jsessionid = loginResult.xhr.getResponseHeader("Set-Cookie").match(/JSESSIONID=[^;]+/)[0];
+
+        var billingStatusData = {
+            "action": "getBillingStatusOfTenant",
+            "tenantDomain": tenantDomain,
+            "cloudType": subscription
+        };
+
+        try {
+            billingResult = post(billingStatusFirstLoginUrl,billingStatusData, {'Cookie':jsessionid+';'}, 'json');
+        } catch(e) {
+            log.error('Error occurred while retrieving billing status data' + e);
+        }
+
+        if(billingResult && billingResult.data && billingResult.data.data) {
+                isPaidAccount = billingResult.data.data.isPaidAccount;
+                billingPlanStatus = billingResult.data.data.billingPlanStatus;
+        } else {
+            log.error('Response mismatch error for billingResult response');
```

Comment:
Make this log more descriptive.Include unable to load billing information for which user or which tenant"